net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild

```diff
@@ -1,28 +1,28 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 PYTHON_COMPAT=( python3_{7..9} )
 PYTHON_REQ_USE=( sqlite )
-DISTUTILS_USE_SETUPTOOLS=rdepend
 
 inherit distutils-r1 optfeature
 
+MY_COMMIT="9b6b0bc1252daa6e574363d87d04eebd981215a5"
 DESCRIPTION="Tool for searching quassel logs from the commandline"
 HOMEPAGE="https://github.com/fish-face/quasselgrep"
-
-MY_COMMIT="9b6b0bc1252daa6e574363d87d04eebd981215a5"
```

Comment:
Imo, either move `MY_COMMIT` before `inherit` where it is usually placed, or keep it here since it is related to `SRC_URI` more than the block above.

Reply:
Well I usually see it placed between `inherit` and `DESCRIPTION`, but its fine for me either way